rebar.config

```diff
@@ -1,2 +1,6 @@
+%% -*- mode: erlang -*-
 {erl_opts, [debug_info]}.
-{deps, [jsx]}.
+{deps, [
+    {jsx, {git, "https://github.com/talentdeficit/jsx", {tag, "v3.1.0"}}},
```

Comment:
no need for jsx because erlavro comes with jsone:
https://github.com/klarna/erlavro/blob/master/rebar.config#L12